nodebuilder/tests/tastora/blob_test.go

```diff
@@ -31,319 +34,432 @@ func TestBlobTestSuite(t *testing.T) {
 
-	libBlob1, err := share.NewV1Blob(namespace, data1, nodeAddr.Bytes())
+	txConfig := state.NewTxConfig(state.WithGas(250_000), state.WithGasPrice(5000))
+	height, err := bridgeClient.Blob.Submit(ctx, nodeBlobs, txConfig)
 	s.Require().NoError(err)
-	libBlob2, err := share.NewV1Blob(namespace, data2, nodeAddr.Bytes())
+
+	// Wait for inclusion
+	_, err = bridgeClient.Header.WaitForHeight(ctx, height)
 	s.Require().NoError(err)
 
-	nodeBlobs, err := nodeblob.ToNodeBlobs(libBlob1, libBlob2)
+	// Allow propagation
+	time.Sleep(10 * time.Second)
+
+	// Ensure light node is synced
+	_, err = lightClient.Header.WaitForHeight(ctx, height)
 	s.Require().NoError(err)
 
-	txConfig := state.NewTxConfig(state.WithGas(400_000), state.WithGasPrice(5000))
-	height, err := client.Blob.Submit(ctx, nodeBlobs, txConfig)
+	// Get proofs from both nodes
+	bridgeProof, err := bridgeClient.Blob.GetProof(ctx, height, namespace, nodeBlobs[0].Commitment)
+	s.Require().NoError(err, "bridge should generate proof")
+
+	lightProof, err := lightClient.Blob.GetProof(ctx, height, namespace, nodeBlobs[0].Commitment)
+	s.Require().NoError(err, "light should generate proof")
+
+	// Verify proof consistency
+	s.Assert().Equal(len(*bridgeProof), len(*lightProof), "proof lengths should match")
+
+	for i, bridgeProofPart := range *bridgeProof {
+		if i < len(*lightProof) {
+			s.Assert().Equal(bridgeProofPart, (*lightProof)[i],
+				"proof part %d should be identical between bridge and light", i)
+		}
+	}
+
+	// Verify inclusion on both nodes
+	bridgeIncluded, err := bridgeClient.Blob.Included(ctx, height, namespace, bridgeProof, nodeBlobs[0].Commitment)
 	s.Require().NoError(err)
+	s.Assert().True(bridgeIncluded, "bridge should verify inclusion")
 
-	// Wait for inclusion
-	_, err = client.Header.WaitForHeight(ctx, height)
+	lightIncluded, err := lightClient.Blob.Included(ctx, height, namespace, lightProof, nodeBlobs[0].Commitment)
 	s.Require().NoError(err)
+	s.Assert().True(lightIncluded, "light should verify inclusion")
 
-	// Test GetAll API
-	retrievedBlobs, err := client.Blob.GetAll(ctx, height, []share.Namespace{namespace})
+	// Cross-verification: bridge proof should work on light node
+	crossIncluded, err := lightClient.Blob.Included(ctx, height, namespace, bridgeProof, nodeBlobs[0].Commitment)
 	s.Require().NoError(err)
-	s.Require().Len(retrievedBlobs, 2)
+	s.Assert().True(crossIncluded, "bridge proof should verify on light node")
 
-	// Verify all blobs belong to correct namespace
-	for _, blob := range retrievedBlobs {
-		s.Assert().True(blob.Namespace().Equals(namespace))
-	}
+	s.T().Logf("✅ Blob proof consistency verified across nodes")
 }
 
-// TestBlobGetProof_ValidBlob tests GetProof API functionality
-func (s *BlobTestSuite) TestBlobGetProof_ValidBlob() {
-	ctx, cancel := context.WithTimeout(context.Background(), 60*time.Second)
+// TestLightNodeBlobAccess validates that light nodes can successfully access
```

Comment:
I'm okay with this test I guess - but there's coverage for all of these cases via other tests.

I'd move it higher up then I guess as a plain sanity check. Also pls rename and change the description as it's confusing.